pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchgit, pkgconfig, cmake, gnumake, glibc
+, enableJpeg ? false, libjpeg
```

Comment:
@7c6f434c , I am also including options as to whether to build documentation or examples as we speak, both of which are not literally mapped to dependency libraries. I'd like to keep the naming congruent and uniform, but if you think it'd be better as a non-uniform naming schema, please let me know and I can change it!